CMakeLists.txt

```diff
@@ -13,8 +13,12 @@
 cmake_minimum_required(VERSION 2.8.4 FATAL_ERROR)
 cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION})
 
+IF (NOT DEFINED KFS_DIR_PREFIX)
+    set (KFS_DIR_PREFIX ${CMAKE_SOURCE_DIR})
```

Comment:
The change reverts part of recent commit efd1d2eb1feb9999414a1e1a61e4fe1ce2decd89 which I do not approve, because it breaks existing functionality, with no clear motivation or gain. I believe that the commit efd1d2eb1feb9999414a1e1a61e4fe1ce2decd89  or part of this commit should be reverted in order to restore broken functionality required by verification and testing. Problem statement and discussion can follow.